core/src/main/java/io/neow3j/protocol/core/response/NativeContractState.java

```diff
@@ -2,51 +2,34 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonSetter;
-import com.fasterxml.jackson.annotation.Nulls;
 import io.neow3j.types.Hash160;
 
 import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class NativeContractState extends CoreContractState {
+public class NativeContractState extends ExpressContractState {
 
-    @JsonProperty("updatehistory")
-    @JsonSetter(nulls = Nulls.AS_EMPTY)
-    private List<Integer> updateHistory = new ArrayList<>();
+    @JsonProperty("id")
+    private BigInteger id;
 
-    public NativeContractState() {
-        super();
-    }
+    @JsonProperty("nef")
+    private ContractNef nef;
 
-    public NativeContractState(BigInteger id, Hash160 hash, ContractNef nef, ContractManifest manifest,
-            List<Integer> updateHistory) {
-
-        super(id, hash, nef, manifest);
-        this.updateHistory = updateHistory;
+    public NativeContractState() {
     }
 
-    @Deprecated
-    public List<Integer> getUpdateHistory() {
-        return updateHistory;
+    public NativeContractState(BigInteger id, Hash160 hash, ContractNef nef, ContractManifest manifest) {
+        super(hash, manifest);
+        this.id = id;
+        this.nef = nef;
     }
 
-    @Override
-    public int hashCode() {
-        return Objects.hash(getId(), getHash(), getNef(), getManifest());
+    public BigInteger getId() {
+        return id;
     }
 
-    @Override
-    public String toString() {
```

Comment:
hmmm... `toString()` not required here? not even for tests, whatsoever?

Reply:
Mhhh, it wasn't there before, so I didn't notice. I'm quickly gonna voice my distaste for the `toString`, `equals`, `hashCode` methods and will add them anyway.

Reply:
Once those are added again it's good to go, imo.

Reply:
Claude dissatisfaction when looking my comment:

![Disappointed Let Down - l41YzNlhKqyJqQZG0](https://github.com/neow3j/neow3j/assets/2124276/8229a157-e955-4e04-828e-d251297c6116)

Reply:
🤣